client/gutenberg/extensions/presets/jetpack/utils/block-category.js

```diff
@@ -0,0 +1,13 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { getCategories, setCategories } from '@wordpress/blocks';
+
+setCategories( [
+	// Add a Jetpack block category
+	{ slug: 'jetpack', title: __( 'Jetpack' ) },
+	...getCategories(),
```

Comment:
might we want to guard against double-inclusion here?

```js
setCategories( [
	...getCategories().filter( ( { slug } ) => slug !== 'jetpack' ),
	{ slug: 'jetpack', title: __( 'Jetpack' ) }
] );
```

I'm thinking that at least early on we could imagine a scenario where a theme relies on Jetpack blocks but then accidentally tries to introduce the Jetpack category again. Maybe this is automatically handled inside Gutenberg 🤔

Reply:
Good point, fixed in https://github.com/Automattic/wp-calypso/pull/27190/commits/2a756e9b5860ee35f9bfbe48cfd15a7d8517bf7f